### ISSUE TITLE
SSH Agent issue, and install.sh has windows line endings

### ISSUE THREAD

**Issue by scrappylabs** [stevelorenz/comnetsemu#19]
Issue 1:
Was attempting to install using Vagrant on Windows with VirtualBox as the provider, kept receiving an error about SSH agent not working. I ended up having to run:
vagrant halt
vagrant destroy -f
vagrant up comnetsemu

When none of those options worked, I ended up having to open VirtualBox, and then running vagrant up commnetsemu, and the SSH error went away and I was able to vagrant ssh comnetsemu

Issue 2:
when running install.sh from ~/comnetsemu/util/
I kept getting an error: 
./install.sh: line 9: $'\r'

Basically, the install.sh script does not have unix endings.
Had to run: sed $'s/\r$//' ./install.sh > ./install.Unix.sh
then I ran: bash ./install.Unix.sh -u 
everything worked...I guess I'm asking to please be careful editing bash files in windows! :)

**Comment by stevelorenz** [comment]
Hi,

Thanks for your feedback.

- Issue 1: The Windows environment is not tested in the CI. I currently don't have the bandwidth to debug and fix bugs for Windows environment support.
- Issue 2: The install script should have the Unix ending. Not sure why Windows somehow changed the endings... 

With best regards
